SimPEG/SEIS/__init__.py

```diff
@@ -0,0 +1 @@
+from . import StraightRay
```

Comment:
do we want to call this `StraightRayTomo` or `StraightRayTomography` or is `StraightRay` descriptive enough?